packages/zoe/src/zoeService/installationStorage.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/prefer-ts-expect-error -- accomodate different type search depths */
```

Comment:
How much type-checking coverage are we losing here?

Reply:
this is just to allow using `ts-ignore` instead of `ts-expect-error`